app/src/main/java/com/android/calculator2/Calculator.java

```diff
@@ -345,6 +349,12 @@ public void onUserInteraction() {
     }
 
     public void onButtonClick(View view) {
+
+        // Play audio for numbers and operators
+        if (view.getTag() != null && !RAW_FILE_EQUALS.equals(view.getTag())) {
```

Comment:
Use `(view.getTag() != null)` to make it more readable.

http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-137265.html#331